tests/testthat/test-test-that.R

```diff
@@ -89,46 +89,46 @@ expectation_lines <- function(code) {
 
 test_that("line numbers captured in simple case", {
   lines <- expectation_lines({
-    context('testing testFile')  # line 1
-    test_that('simple', {        # line 2
-      expect_true(FALSE)         # line 3
-    })                           # line 4
+    context("testing testFile") # line 1
```

Comment:
Can you manually revert the changes to this file? The indenting of the comments is particularly important here